packages/web/src/components/track/PlayPauseButton.tsx

```diff
@@ -1,33 +1,62 @@
-import { FeatureFlags } from '@audius/common'
+import { FeatureFlags, PlaybackStatus } from '@audius/common'
 import { Button, ButtonType, IconPause, IconPlay } from '@audius/stems'
 
+import { ReactComponent as IconRepeat } from 'assets/img/iconRepeatOff.svg'
 import { useFlag } from 'hooks/useRemoteConfig'
 
 import styles from './GiantTrackTile.module.css'
 
 type PlayPauseButtonProps = {
   doesUserHaveAccess: boolean
   playing: boolean
+  playbackStatus?: PlaybackStatus | null
   onPlay: () => void
 }
 
+const messages = {
+  play: 'PLAY',
```

Comment:
nice. small nit which was there before, is that we could do `play` as the string and do `textTransform: uppercase`, that is technically more accessible, since screen reader would just read "P""L""A""Y"

Reply:
🧠